src/fsharp/symbols/Symbols.fs

```diff
@@ -2019,6 +2019,35 @@ and FSharpMemberOrFunctionOrValue(cenv, d:FSharpMemberOrValData, item) =
         | V valRef -> not (SymbolHelpers.isFunction cenv.g valRef.Type)
         | _ -> false
 
+    member x.IsComputationExpressionMethod =
+        let isNameN (methodName: string) (methodKind: string) =
+            if methodName.StartsWith(methodKind) then
+                let (didParse, _) = System.Int32.TryParse(methodName.[methodKind.Length..])
+                didParse
+            else
+                false
+        let isBindNReturn (methodName: string) =
+            if methodName.StartsWith("Bind") && methodName.EndsWith("Return") then
+                let (didParse, _) = System.Int32.TryParse(methodName.[4..methodName.Length-7]) // replace this disgusting mess with reverse indexing some time
```

Comment:
I'd like to see a test, particularly for this monstrosity.

Reply:
I hate so much the giant baby

Reply:
Ouch. There no alternative way around this?

Reply:
Gotta find the number between `Bind` and `Return` in `Bind123Return`, so yeah I think it is. I guess I could pull out some values into names to help